rs/crypto/internal/crypto_lib/multi_sig/bls12_381/src/api.rs

```diff
@@ -108,7 +108,9 @@ pub fn verify_individual(
 ) -> Result<(), CryptoError> {
     let signature = IndividualSignature::try_from(signature_bytes)?;
     let public_key = PublicKey::try_from(public_key_bytes)?;
-    if crypto::verify_individual_message_signature(message, &signature, &public_key) {
+    if crypto::verify_individual_message_signature(message, &signature, &public_key)
+        && !public_key.is_identity()
```

Comment:
Should also add this check in `verify_combined`?

Also, how about adding tests ensuring that identity-public-keys are rejected now (just like in https://github.com/dfinity/vetkeys/pull/141)?

Reply:
I could, but I realized after that first PR that these tests are pretty fake - they fail regardless if the check for the identity public key is there or not. It is not afaict possible to create a test that fails without the check and passes with the check added, because it is impossible to recover the discrete logarithm of the output of hash2curve.